projects/demo/src/app/dropdown/dropdown-angular-nested.demo.html

```diff
@@ -73,4 +73,34 @@
       <div clrDropdownItem>Link 2</div>
     </clr-dropdown-menu>
   </clr-dropdown>
+
+  <div class="content-area" style="position: relative">
+    <div style="margin-top: 800px"></div>
```

Comment:
Reduce the `top-margin` to make the dropdown visible or show in some other way that there is something at the bottom of the page.

```suggestion
    <div style="margin-top: 400px"></div>
```